mne/defaults.py

```diff
@@ -11,22 +11,26 @@
                ref_meg='steelblue', misc='k', stim='k', resp='k', chpi='k',
                exci='k', ias='k', syst='k', seeg='saddlebrown', dipole='k',
                gof='k', bio='k', ecog='k', hbo='#AA3377', hbr='b',
-               fnirs_cw_amplitude='k', fnirs_od='k', csd='k'),
+               fnirs_cw_amplitude='k', fnirs_fd_ac_amplitude='k',
```

Comment:
Would it be good to use different color for the different returned types? I am thinking that users will start by simply plotting their returned signals and want to see that three types were returned, maybe different colors would make it easier to see the different types and notice the different patterns in each type.

Reply:
yes sure, what other colours are available for phase and ac data?

Reply:
Is there a standard in the literature (similar to how HbO is always red and HbR blue?), otherwise something pleasing (anything but fluro green please ;) )